cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -164,7 +169,11 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
 
         outputFiles.forEach { file ->
             println("Writing analyzer result to '$file'.")
-            file.mapper().writerWithDefaultPrettyPrinter().writeValue(file, ortResult)
+            val duration = measureTime { file.mapper().writerWithDefaultPrettyPrinter().writeValue(file, ortResult) }
+
+            log.perf {
+                "Wrote ORT result to ${file.name} (${file.sizeInMiB}) in ${duration.inMilliseconds}ms."
```

Comment:
Nit: I believe we usually single-quote file names / paths, and your proposal originally also included those.

Reply:
Similar below.

Reply:
Done.